### DISCUSSION TITLE
Cannot run code in python

### DISCUSSION THREAD

**Discussion by Lemon553311-dev** [coder/code-server#3350] [Q&A]
Just installed code-server and when trying to run code it just gives `command 'python.execInTerminal-icon' not found` error in the bottom right. how can I fix that ?

**Comment by jsjoeio** [comment]
Hmm...there may be a bug. I tried running a Python script just now and had no issues

![image](https://user-images.githubusercontent.com/3806031/117867445-3e288180-b24d-11eb-9a46-0c0c625c4629.png)

Can you open a [bug report](https://github.com/cdr/code-server/issues/new?assignees=&labels=&template=bug-report.md&title=) please?
